cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -90,7 +87,7 @@ define([
         delete window.course;
     });
 
-    describe('GroupConfigurationDetails', function() {
+    describe('GroupConfigurationDetailsView', function() {
```

Comment:
I think clarity is more important than following existing patterns (which is actually a pretty lame pattern anyway). Please just change the string to be clear, could be something like 

"GroupConfigurationsDetailsView, the details view for Experiment group configurations"